### PR TITLE
[FJD-47] Configure additional blocks

### PR DESCRIPTION
**Pull request by david-storm**
Jira issue:

Steps to review:
- [ ] login as admin
- [ ] Create a basic page
- [ ] Go to layout this page
- [ ] Click **add block**
Ensure that you can create following blocks:
- [ ] Image Carousel
- [ ] Banner
- [ ] Video
- [ ] Gallery (Photo/Video gallery)
- [ ] Webforms

### CODE REVIEW

File: config/install/core.entity_form_display.block_content.carousel_item.default.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    region: content
+    settings:
+      entity_browser: media_directories_modal
+      field_widget_display: rendered_entity
+      field_widget_edit: true
+      field_widget_remove: true
+      field_widget_replace: false
+      open: true
+      field_widget_display_settings:
+        view_mode: default
```
Comment (OleksandrRiumshyn) [inline config/install/core.entity_form_display.block_content.carousel_item.default.yml:38]: use some separate view mode for thumbnail. for example media library

File: config/install/core.entity_form_display.block_content.carousel.default.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+id: block_content.carousel.default
+targetEntityType: block_content
+bundle: carousel
+mode: default
+content:
+  field_item:
+    type: inline_entity_form_complex
+    weight: 26
+    region: content
+    settings:
```
Comment (OleksandrRiumshyn) [inline config/install/core.entity_form_display.block_content.carousel.default.yml:19]: after adding items Block description is empty(looks like a bug)

File: config/install/core.entity_form_display.block_content.carousel.default.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    - block_content.type.carousel
+    - field.field.block_content.carousel.field_item
+  module:
+    - inline_entity_form
+id: block_content.carousel.default
+targetEntityType: block_content
+bundle: carousel
+mode: default
+content:
+  field_item:
```
Comment (OleksandrRiumshyn) [inline config/install/core.entity_form_display.block_content.carousel.default.yml:15]: field is multiple, so I think we need better name. Maybe items or children

File: config/install/field.field.block_content.carousel_item.field_image.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+uuid: 5dc82726-9fa9-4aa0-a5e4-e0652582b827
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.carousel_item
+    - field.storage.block_content.field_image
+    - media.type.image
+id: block_content.carousel_item.field_image
```
Comment (OleksandrRiumshyn) [inline config/install/field.field.block_content.carousel_item.field_image.yml:9]: (optional) I think you could reuse media field

File: config/install/field.field.block_content.video.field_video.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+uuid: a9dcdecd-cecd-4492-b1b1-95b2ac5d1a09
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.video
+    - field.storage.block_content.field_video
+    - media.type.video
+id: block_content.video.field_video
```
Comment (OleksandrRiumshyn) [inline config/install/field.field.block_content.video.field_video.yml:9]: (optional) I think you could reuse media field

File: config/install/field.storage.block_content.field_media.yml [hunk 6]
```diff
@@ -0,0 +1,20 @@
+uuid: 7ce04334-3bd0-49c2-8982-599f09ac88dd
+langcode: en
+status: true
+dependencies:
+  module:
+    - block_content
+    - media
+id: block_content.field_media
```
Comment (OleksandrRiumshyn) [inline config/install/field.storage.block_content.field_media.yml:8]: The field is multiple so naming is no correct. Use please plural version instead ("medias")

File: config/install/field.storage.block_content.field_video.yml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+uuid: 578ca695-f820-417b-a049-5460b504d5f4
+langcode: en
+status: true
+dependencies:
+  module:
+    - block_content
+    - media
+id: block_content.field_video
```
Comment (OleksandrRiumshyn) [inline config/install/field.storage.block_content.field_video.yml:8]: Could be combined with image field to one storage field_media

File: config/install/block_content.type.carousel_item.yml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+uuid: 8b96c671-eaa2-4d18-a6f0-d558b9f2ce73
+langcode: en
+status: true
+dependencies: {  }
+id: carousel_item
```
Comment (OleksandrRiumshyn) [inline config/install/block_content.type.carousel_item.yml:5]: I'm not sure that we need block type here, maybe to use paragraphs would be easier solution. Let's discuss it together at Team sync call

**Review by OleksandrRiumshyn — CHANGES_REQUESTED** [review]

File: config/install/core.entity_form_display.block_content.carousel.default.yml (hunk 2 above)
Comment (david-storm) [reply config/install/core.entity_form_display.block_content.carousel.default.yml:18]: I added block description to form dispaly

**Review by OleksandrRiumshyn — APPROVED** [review]

**Comment by OleksandrRiumshyn** [comment]
Merged with other related branch

**Pull request closed without merging**